pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "nucypher"
+version = "0.1.0-alpha.0"
+description = "A proxy re-encryption network to empower privacy in decentralized systems."
+authors = ["NuCypher"]
+license = "GPL-3.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+
+#
+# NuCypher
+#
+umbral = "==0.1.2a2"
```

Comment:
We should be using `==0.1.3a0`. I'm afraid this file may be inconsistent with current Pipfile on master.

Reply:
Perhaps we should remove poetry config for the moment. Later on we can include these files in a dedicated PR, including changes to integrate it with CI

Reply:
:checkered_flag: PR #622

Reply:
You're fast, man!